packages/components/psammead-topic-tags/src/index.stories.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs } from '@storybook/addon-knobs';
+import {
+  withServicesKnob,
+  buildRTLSubstories,
+} from '@bbc/psammead-storybook-helpers';
+import { TopicTags, TopicTag } from '.';
+import notes from '../README.md';
+
+const STORY_KIND = 'Components/TopicTags';
+
+storiesOf(STORY_KIND, module)
+  .addDecorator(withKnobs)
+  .addDecorator(withServicesKnob())
+  .add(
+    'default',
+    ({ dir, service, script, text }) => {
+      const shortText = service === 'news' ? text : text.trim().split(' ')[0];
+
+      return (
+        <TopicTags dir={dir} service={service} script={script}>
+          <TopicTag topicName={shortText} topicLink="#" />
+        </TopicTags>
+      );
+    },
+    { notes, knobs: { escapeHTML: false } },
+  )
+  .add(
+    'multiple topics',
+    ({ dir, service, script, text }) => {
+      const shortText = service === 'news' ? text : text.trim().split(' ')[0];
+
+      return (
+        <TopicTags dir={dir} service={service} script={script}>
+          <TopicTag topicName={shortText} topicLink="#" />
+          <TopicTag topicName={shortText} topicLink="#" />
+          <TopicTag topicName={shortText} topicLink="#" />
+          <TopicTag topicName={shortText} topicLink="#" />
+        </TopicTags>
+      );
+    },
+    { notes, knobs: { escapeHTML: false } },
+  );
+
+buildRTLSubstories(STORY_KIND, { include: ['default', 'multiple topics'] });
```

Comment:
Is there a reason why we're adding separate RTL stories when the RTL services can just be accessed through the other stories?

Reply:
Hey Harvey, thanks for the comments!

I must have instinctively used this method after the consent banner work. Happy to remove it 👍